profiles/common/modules/custom/nexteuropa_ecl_migrate/nexteuropa_ecl_migrate.module

```diff
@@ -218,3 +227,84 @@ function _nexteuropa_ecl_migrate_remove_blocks_header_top() {
   multisite_drupal_toolbox_remove_block_context('site_wide', 'menu-service-tools');
   multisite_drupal_toolbox_remove_block_context('site_wide', 'language_selector_site');
 }
+
+/**
+ * Custom function that moves blocks from the nexteuropa_community feature.
+ */
+function _nexteuropa_ecl_migrate_move_community_blocks() {
+  if (module_exists('nexteuropa_communities')) {
```

Comment:
I noticed that in the other methods we remove the blocks from their original location first. Shouldn't we do this here as well?

Reply:
@imanoleguskiza yes i do this here : 
multisite_drupal_toolbox_remove_block_context
Or am I missing a point.

Reply:
As discussed this is now modified so it's consistent, for optional features we only remove if the feature is enabled.